src/api/vehicles/client/setVehicleProperties.lua

```diff
@@ -4,213 +4,214 @@
 ---@return void
 ---@public
 function _NCS:setVehicleProperties(vehicleEntity, props)
-    if DoesEntityExist(vehicleEntity) then
-        local colorPrimary, colorSecondary = GetVehicleColours(vehicleEntity)
-        local pearlescentColor, wheelColor = GetVehicleExtraColours(vehicleEntity)
-        SetVehicleModKit(vehicleEntity, 0)
+    if not DoesEntityExist(vehicleEntity) then return _NCS:trace("Can't set vehicle properties for the vehicle (entity doesn't exist)") end
+    if NetworkGetEntityOwner(vehicleEntity) ~= PlayerId() then return _NCS:trace("Can't set vehicle properties for the vehicle (client is not the entity owner") end
```

Comment:
Please consider using _NCS#die introduced in this commit https://github.com/NextCitizens/ncs_core/commit/53bbdf825e1530fd24e0207bf7bdfcb1179f1a5c

Reply:
Should be good now